docker-vertica/README.md

```diff
@@ -4,25 +4,10 @@
 * Docker Community: [Docker Forums](https://forums.docker.com/), [Stack Overflow](https://stackoverflow.com/questions/tagged/docker)
 
 # Supported Tags
-* [12.0.2-0-minimal, latest](https://github.com/vertica/vertica-kubernetes/blob/v1.8.0/docker-vertica/Dockerfile)
-* [12.0.2-0](https://github.com/vertica/vertica-kubernetes/blob/v1.8.0/docker-vertica/Dockerfile)
-* [12.0.1-0-minimal](https://github.com/vertica/vertica-kubernetes/blob/v1.7.0/docker-vertica/Dockerfile)
-* [12.0.1-0](https://github.com/vertica/vertica-kubernetes/blob/v1.7.0/docker-vertica/Dockerfile)
-* [12.0.0-0-minimal](https://github.com/vertica/vertica-kubernetes/blob/v1.6.0/docker-vertica/Dockerfile)
-* [12.0.0-0](https://github.com/vertica/vertica-kubernetes/blob/v1.6.0/docker-vertica/Dockerfile)
-* [11.1.1-2-minimal](https://github.com/vertica/vertica-kubernetes/blob/v1.5.0/docker-vertica/Dockerfile)
-* [11.1.1-2](https://github.com/vertica/vertica-kubernetes/blob/v1.5.0/docker-vertica/Dockerfile)
-* [11.1.1-0-minimal](https://github.com/vertica/vertica-kubernetes/blob/v1.4.0/docker-vertica/Dockerfile)
-* [11.1.1-0](https://github.com/vertica/vertica-kubernetes/blob/v1.4.0/docker-vertica/Dockerfile)
-* [11.1.0-0-minimal](https://github.com/vertica/vertica-kubernetes/blob/v1.3.0/docker-vertica/Dockerfile)
-* [11.1.0-0](https://github.com/vertica/vertica-kubernetes/blob/v1.3.0/docker-vertica/Dockerfile)
-* [11.0.2-0-minimal](https://github.com/vertica/vertica-kubernetes/blob/v1.2.0/docker-vertica/Dockerfile)
-* [11.0.2-0](https://github.com/vertica/vertica-kubernetes/blob/v1.2.0/docker-vertica/Dockerfile)
-* [11.0.1-0-minimal](https://github.com/vertica/vertica-kubernetes/blob/v1.1.0/docker-vertica/Dockerfile)
-* [11.0.1-0](https://github.com/vertica/vertica-kubernetes/blob/v1.1.0/docker-vertica/Dockerfile)
-* [11.0.0-0-minimal](https://github.com/vertica/vertica-kubernetes/blob/v1.0.0/docker-vertica/Dockerfile)
-* [11.0.0-0](https://github.com/vertica/vertica-kubernetes/blob/v1.0.0/docker-vertica/Dockerfile)
-* [10.1.1-0](https://github.com/vertica/vertica-kubernetes/blob/v0.1.0/docker-vertica/Dockerfile)
+
+The [VerticaDB operator](https://hub.docker.com/r/vertica/verticadb-operator) installs the `vertica-k8s` image with the most recently released patch version and _-minimal_ tag.
```

Comment:
The VerticaDB will only provide a default image only if one wasn't provided in the VerticaDB Custom Resource (CR). I would just keep this part a description of how the tag is selected.

```suggestion
We tag each container with the Vertica version. Each version is provided in two formats: a full version and a minimal version. The minimal version has some things stripped out to reduce its size. Refer to the [documentation](https://www.vertica.com/docs/latest/HTML/Content/Authoring/Containers/Images.htm?tocpath=Containerized Vertica|_____2) for a detailed description of the differences.

Each tag follows this format: '_<major>.<minor>.<patch>-<hotfix>[-minimal]_'. 

Some examples:
- `11.1.0-0`: is the tag for the full image of 11.1, service pack 0, hotfix 0.
- `12.0.1-0-minimal`: is the tag for the minimal image of 12.0, service pack 1, hotfix 0.

The tag `latest` always refers to the minimal image of the most recently released version.
```